AEPServices/Sources/ui/fullscreen/FullscreenMessageDelegate.swift

```diff
@@ -34,6 +34,10 @@ import Foundation
     @objc(overrideUrlLoadFullscreenMessage:url:)
     func overrideUrlLoad(message: FullscreenMessage, url: String?) -> Bool
 
+    /// Invoked when the fullscreen message finished loading the webView.
+    @objc(webViewDidFinishLoading)
+    optional func webViewDidFinishLoading()
```

Comment:
In iOS we typically pass in the object that is being referenced, so I think I'd like to see something like:
`webViewDidFinishLoading(webView: WKWebView)` for this protocol definition.

For example, if you take a look at the `UITableViewDelegate` you will notice that the `UITableView` in reference is nearly always passed into the delegate function.
https://developer.apple.com/documentation/uikit/uitableviewdelegate